.github/workflows/tests.yml

```diff
@@ -54,9 +54,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: actions/setup-python@v4
+    - uses: actions/setup-python@v5
       with:
         python-version: "3.11"
 
     - run: pip install -U pip tox
-    - run: tox -e docs,style,security
+    - run: tox -e docs,style
```

Comment:
Will use `ruff` instead of a dedicated `security` tox target (which was using `bandit`)